fluent-bit-kinesis.go

```diff
@@ -66,6 +75,12 @@ func newKinesisOutput(ctx unsafe.Pointer, pluginID int) (*kinesis.OutputPlugin,
 	logrus.Infof("[firehose %d] plugin parameter time_key = '%s'\n", pluginID, timeKey)
 	timeKeyFmt := output.FLBPluginConfigKey(ctx, "time_key_format")
 	logrus.Infof("[firehose %d] plugin parameter time_key_format = '%s'\n", pluginID, timeKeyFmt)
+	concurrency := output.FLBPluginConfigKey(ctx, "concurrency")
+	logrus.Infof("[firehose %d] plugin parameter concurrency = '%s'\n", pluginID, concurrency)
+	concurrencyRetries := output.FLBPluginConfigKey(ctx, "concurrency_retries")
+	logrus.Infof("[firehose %d] plugin parameter concurrency_retries = '%s'\n", pluginID, concurrencyRetries)
+	aggregated := output.FLBPluginConfigKey(ctx, "aggregated")
+	logrus.Infof("[firehose %d] plugin parameter aggregated = '%s'\n", pluginID, aggregated)
```

Comment:
aggregated? I think this doesn't belong in this PR...

(Though if this means you're separately also working on KPL format, I am very happy 😃)

Reply:
Yes.  Oops, I'll remove that.  I missed that change when switching contexts.